### DISCUSSION TITLE
菜单图标不显示，菜单文字对齐和与动画不一致，瞬间无法设置封面图，以及中英文切换

### DISCUSSION THREAD

**Discussion by JCVICTORSUN** [LIlGG/halo-theme-sakura#385] [General]
### 目前使用的 Sakura 版本？

2.0.0-beta.2

### 目前使用的 halo 版本？

2.7.0

### 建议/问题

嘛都是一些细节上的问题我详细说明一下：
### 1. 菜单图标不显示
![image](https://github.com/LIlGG/halo-theme-sakura/assets/53136539/4443df19-692c-4740-9bcc-6c40bf8ad84f)
如图我有两个菜单，瞬间是直接菜单新建的/moments，关于是页面新建
![image](https://github.com/LIlGG/halo-theme-sakura/assets/53136539/f3ce3ff4-7f3d-46c2-9ee2-bc1a46502060)
两个菜单的图标无论是SVG代码还是iconify都无法显示
![image](https://github.com/LIlGG/halo-theme-sakura/assets/53136539/7551a18c-86d2-494a-bfa3-53a5963c4813)
### 2. 菜单文字对齐和与动画不一致
如果，我直接新建了一个菜单
![image](https://github.com/LIlGG/halo-theme-sakura/assets/53136539/80f9658e-7982-4b2f-bafe-e90d6d4d951a)
主页显示会变成这样：
![image](https://github.com/LIlGG/halo-theme-sakura/assets/53136539/9251c0a4-4d84-406a-8ab8-b7e9e865d587)
但如果说我添加了图标哪怕我之后再把这个图标代码删除，就会变成相同间隔如1.的最后一张图
但是但是，鼠标悬停时的蓝色下划线有问题：直接添加没有图标是这样的（正常）
![image](https://github.com/LIlGG/halo-theme-sakura/assets/53136539/185caa8b-104a-4595-bdd3-52472a5c8a05)
但是一旦添加了图标哪怕再删除就会变成这样：
![image](https://github.com/LIlGG/halo-theme-sakura/assets/53136539/f0cd67ef-f6c6-4ff3-a9e6-fde678fc031d)
也就是说，要不然间隔不对，要不然就是间隔对了但是图标不显示且鼠标悬停时下划线有偏移
### 3. 瞬间无法设置封面图
我尝试了两种添加瞬间页面的方法：第一种就是直接在菜单里添加页面设置为/moments，第二种是在页面添加：
![image](https://github.com/LIlGG/halo-theme-sakura/assets/53136539/f49befe3-5636-458f-a4e5-f30077dd1e86)
首先页面添加可以设置封面图但是对于瞬间没用，无法显示，在菜单里添加更是没有封面图这个选项，希望后续可以添加一下，默认的这个大括号实在是有点丑）
![image](https://github.com/LIlGG/halo-theme-sakura/assets/53136539/0766b6c8-09b9-49a0-b31c-32b200236d07)
还想再问的就是瞬间的图片什么时候可以支持QAQ我这里甚至背景都没有，还是说我创建瞬间的方式有问题）
![image](https://github.com/LIlGG/halo-theme-sakura/assets/53136539/b06d4c52-b808-41a4-be77-bb2c82f40433)
![image](https://github.com/LIlGG/halo-theme-sakura/assets/53136539/9142f276-688c-431a-998b-a64269adb111)

### 4. 中英文切换是为什么被移除了呢）很想做中英文可以互换的网站因为自己是留学生）
以上！阿里嘎多！

**Comment by LIlGG** [comment]
### 菜单文字对齐和与动画不一致
如果，我直接新建了一个菜单
![image](https://github.com/LIlGG/halo-theme-sakura/assets/53136539/80f9658e-7982-4b2f-bafe-e90d6d4d951a)
主页显示会变成这样：
![image](https://github.com/LIlGG/halo-theme-sakura/assets/53136539/9251c0a4-4d84-406a-8ab8-b7e9e865d587)
但如果说我添加了图标哪怕我之后再把这个图标代码删除，就会变成相同间隔如1.的最后一张图
但是但是，鼠标悬停时的蓝色下划线有问题：直接添加没有图标是这样的（正常）
![image](https://github.com/LIlGG/halo-theme-sakura/assets/53136539/185caa8b-104a-4595-bdd3-52472a5c8a05)
但是一旦添加了图标哪怕再删除就会变成这样：
![image](https://github.com/LIlGG/halo-theme-sakura/assets/53136539/f0cd67ef-f6c6-4ff3-a9e6-fde678fc031d)
也就是说，要不然间隔不对，要不然就是间隔对了但是图标不显示且鼠标悬停时下划线有偏移

**Comment by LIlGG** [comment]
。不小心把回答给删除了，我重新回答一遍吧。

1. 菜单图标不显示
   菜单图标使用下述方式就可以
<img width="680" alt="image" src="https://github.com/LIlGG/halo-theme-sakura/assets/31335418/445b3c8c-381f-4170-84f3-ca5f8dab1b1e">

2. 菜单文字对齐和与动画不一致
确实是个问题，已经打开一个新的 issue #382 进行跟踪。

3. 瞬间无法设置封面图
在 `主题设置 - 头部装饰图` 中设置
<img width="601" alt="image" src="https://github.com/LIlGG/halo-theme-sakura/assets/31335418/e6a7515c-38f0-47e7-82e8-4d34a1e71197">

瞬间的九宫格图片后续会支持，目前只支持直接在瞬间内容中的图片。

瞬间背景与图片没有关系，需要在 `主题设置 - 瞬间` 页面进行修改。

4. 中英文切换是为什么被移除了呢
目前版本中会跟随用户的浏览器属性进行自动切换，在不同的用户代理下展示的语言也就不同，因此手动切换的功能就没有太大的必要了。

**Comment by JCVICTORSUN** [comment]
非常感谢！！！(*╹▽╹*)
